Core/src/org/sleuthkit/autopsy/report/modules/portablecase/PortableCaseReportModule.java

```diff
@@ -103,6 +111,12 @@
     private String caseName = "";
     private File caseFolder = null;
     private File copiedFilesFolder = null;
+    
+    // Stores whether files are present in the file repository
+    private enum FileRepoState {
+        EXISTS, DOES_NOT_EXIST;
+    }
+    private final Map<Long, FileRepoState> fileRepositoryLookupCache = new HashMap<>();
```

Comment:
Are we not going to deal with if the state changes? Also, if the FileRepo is redirected to another server in the settings, this cache needs to be invalidated. Will this PortableCase object be GC'd between Case close and open?

Reply:
I just forgot to add clearing the cache to the cleanup() method. That will take care of it.